deploy/bundle/Makefile

```diff
@@ -24,6 +24,9 @@ manifest:
 	TEMPLATE=`mktemp` ; \
 	./generate-csv-template.py > $${TEMPLATE} ; \
 	oc process --local -o yaml --raw=true IMAGE=$(OPERATOR_IMAGE) IMAGE_TAG=$(OPERATOR_IMAGE_TAG) VERSION=$(VERSION) REPLACE_VERSION=$(REPLACE_VERSION) -f $${TEMPLATE} > $(CSV)
+	@if [ "${REPLACE_VERSION}" == "" ]; then \
+	  sed -i.bak "/replaces/d" $(CSV); \
```

Comment:
Why do you need to create a copy? you can do an in-place edit using `-i ""`

Reply:
on Mac, the -i REQUIRES a copy to be made.

Reply:
Actually, which we'll need to delete.  *sigh*